examples/gitlab/build-two-artifacts.gitlab-ci.yml

```
@@ -0,0 +1,130 @@
+  - deploy
+
+include:
+  - project: 'Northern.tech/Mender/mender-ci-workflows'
+    file: 'templates/gitlab/mender-artifact-build.gitlab-ci.yml'
+    ref: 'master'
+  - project: 'Northern.tech/Mender/mender-ci-workflows'
+    file: 'templates/gitlab/mender-artifact-upload.gitlab-ci.yml'
+    ref: 'master'
+  - project: 'Northern.tech/Mender/mender-ci-workflows'
+    file: 'templates/gitlab/mender-deployment-create.gitlab-ci.yml'
+    ref: 'master'
+
+variables:
+  MENDER_DEVICES_GROUP_NAME_ONE: group-one # The name of Mender devices group a deployment with Mender Artifact name `MENDER_ARTIFACT_NAME_ONE` will be triggered to
+  MENDER_DEVICES_GROUP_NAME_TWO: group-two # The name of Mender devices group a deployment with Mender Artifact name `MENDER_ARTIFACT_NAME_TWO` will be triggered to
+  MENDER_ARTIFACT_NAME_ONE: artifact_one.mender # First Mender Artifact file name in the `MENDER_ARTIFACTS_FOLDER` folder
+  MENDER_ARTIFACT_NAME_TWO: artifact_two.mender # Second Mender Artifact file name in the `MENDER_ARTIFACTS_FOLDER` folder
+  MENDER_RELEASE_NAME_ONE: auth_keys-1.0 # The name of Mender release for Mender Artifact name `MENDER_ARTIFACT_NAME_ONE`
+  MENDER_RELEASE_NAME_TWO: some_config-1.0 # The name of Mender release for Mender Artifact name `MENDER_ARTIFACT_NAME_TWO`
+
+#
+# Builds Mender Artifact using 'mender-artifact' tool,
+# generate and save deployment name and deployment group
+#
+build:mender-artifact:one:
+  stage: build
+  extends: .mender:build:artifact
+  script:
+    # create Mender artifact
+    - echo /home/root/.ssh > dest_dir
+    - echo authorized_keys > filename
+    - echo key > authorized_keys
+    - |
+      mender-artifact write module-image \
+        -T single-file \
+        --device-type raspberrypi4 \
+        -o ${MENDER_ARTIFACTS_FOLDER}/${MENDER_ARTIFACT_NAME_ONE} \
+        -n ${MENDER_RELEASE_NAME_ONE} \
+        --software-name auth_keys \
+        --software-version 1.0 \
+        -f dest_dir \
+        -f filename \
+        -f authorized_keys
+
+#
+# Builds Mender Artifact using 'mender-artifact' tool,
+# generate and save deployment name and list of devices to be deployed to
+#
+build:mender-artifact:two:
+  stage: build
+  extends: .mender:build:artifact
+  script:
+    # create Mender artifact
+    - echo /home/root/.ssh > dest_dir
+    - echo authorized_keys > filename
+    - echo key > authorized_keys
+    - |
+      mender-artifact write module-image \
+        -T single-file \
+        --device-type raspberrypi4 \
+        -o ${MENDER_ARTIFACTS_FOLDER}/${MENDER_ARTIFACT_NAME_TWO} \
+        -n ${MENDER_RELEASE_NAME_TWO} \
+        --software-name authorized_keys \
+        --software-version 1.0 \
+        -f dest_dir \
+        -f filename \
+        -f authorized_keys
+
+#
+# Uploads built Mender Artifacts
+#
+publish:mender-artifact:one:
+  stage: publish
+  needs:
+    - build:mender-artifact:one
+  extends: .mender:upload:artifact
+  rules:
+    - if: '$CI_COMMIT_BRANCH =~ /^(main)$/'
```

For this example, should we use the same rule as for the next example?
```suggestion
    - if: '$CI_COMMIT_BRANCH == "main"'
```